README.md

```diff
@@ -182,11 +182,69 @@ In addition to basic REST requests, the Brainstem.js model `fetch` method suppor
 ```javascript
 Post = require('./models/post');
 
-new Post({ id: 1 }).fetch({ include: ['user', 'comments'] })
+new Post({ id: 1 })
+  .fetch({ include: ['user', 'comments'] })
   .done(/* handle result */)
   .fail(/* handle error */);
 ```
 
+Brainstem also supports requesting deeply nested associations. Under the hood, Brainstem makes recursive requests for each layer of associations. There are three ways to make this kind of request, using JSON Objects, Backbone Collections or BrainstemParams. Using Backbone Collections or BrainstemParams enables expressing additional options for the nested includes / associations.
+
+##### JSON Object Example
```

Comment:
Can you not do filters on JSON objects?

Reply:
Does this require actual JSON, or can it just be a POJO? I thought JSON had stricter formatting requirements, like keys being strings, using double quotes, etc.

Reply:
ah, yeah I will change this to Objects. JSON-like *

Reply:
No, you cannot use filters. That was a large reason this effort was introduced.